river/compose/union.py

```diff
@@ -118,6 +120,35 @@ class TransformerUnion(base.Transformer):
     ...     ('# by place', count)
     ... )
 
+    Mini-batch example:
+
+    >>> X = [
+    ... {"place": 2, "revenue": 42},
+    ... {"place": 3, "revenue": 16},
+    ... {"place": 3, "revenue": 24},
+    ... {"place": 2, "revenue": 58},
+    ... {"place": 3, "revenue": 20},
+    ... {"place": 2, "revenue": 50},
+    ... ]
+
+    Since we need a transformer with mini-batch support to demonstrate, we shall use the `StandardScaler`
+
+    >>> from river import compose
+    >>> from river import preprocessing
+
+    >>> t_1 = compose.Select("place") | preprocessing.StandardScaler()
+    >>> t_2 = compose.Select("revenue") | preprocessing.StandardScaler()
```

Comment:
For the sake of example, I would use a different scaler for `t_2`

Reply:
`StandardScaler` is the only scaler that supports `learn_many`

To keep it concise, could do instead:
```python
t_1 = compose.Select("place") 
t_2 = compose.Select("revenue") | preprocessing.StandardScaler()
```

What do you think?

Reply:
> StandardScaler is the only scaler that supports learn_many

*facepalm*

> What do you think?

Sure that's a good idea :)